components/video-recorder/src/HeadlessVideoRecorder.tsx

```diff
@@ -0,0 +1,636 @@
+import React, {
+	useRef,
+	useState,
+	CSSProperties,
+	useCallback,
+	useEffect,
+	useImperativeHandle,
+} from "react";
+import { twMerge } from "tailwind-merge";
+import { useTimer } from "@locoworks/reusejs-toolkit-react-hooks";
+export interface HeadlessVideoRecorderRef {
+	recording: string;
+	handleDownload: () => void;
+	showPreview: () => void;
+}
+interface HeadlessVideoRecorderProps {
+	autoStop?: boolean;
+	autoPreview?: boolean;
+	timeInMs?: number;
+	handleStateChange?: (state: VideoState) => void;
+	customhandleDownload?: (file: any) => void;
+	customCountDown?: (count: number) => React.ReactNode;
+	customRetakeButton?: (onRetake: any) => React.ReactNode;
```

Comment:
Use the type of function instead of any.

Reply:
Changed to -
	handleStateChange?: (state: VideoState) => void;
	customhandleDownload?: (file: File | null) => void;
	customCountDown?: (count: number) => React.ReactNode;
	customRetakeButton?: (onRetake: () => void) => React.ReactNode;